Scripts/RemediationScripts/Remediate-EnableHttpsForStorageAccount.ps1

```diff
@@ -0,0 +1,723 @@
+﻿<##########################################
```

Comment:
Please fix the script name to reflect the Control Name (and not the property being evaluated). Also, the remediation/roll back commands need to be modified (if required) to reflect this change.